lte/gateway/docker/README.md

```diff
@@ -40,6 +40,7 @@ containerized AGW by running the following steps inside the VM:
 ```
 cd $MAGMA_ROOT/lte/gateway && make run  # You can skip this if you have built the AGW with make before
 for component in redis nghttpx td-agent-bit; do cp "${MAGMA_ROOT}"/{orc8r,lte}/gateway/configs/templates/${component}.conf.template; done
+# Optional: cp ${MAGMA_ROOT}/.cache/test_certs/rootCA.pem /var/opt/magma/certs/
```

Comment:
It might be helpful if this gave a bit more info about when it should be copied

Reply:
Maybe
`If you want to connect to an orc8r, copy the rootCA.pem from the orc8r to /var/opt/magma/certs/rootCA.pem. For example in a typical magma-dev VM: cp ${MAGMA_ROOT}/.cache/test_certs/rootCA.pem /var/opt/magma/certs/`

Reply:
I would say because the volumes are mounted, it doesn't really matter when it is copied, so adding it after the code block to start the containers as an option should be sufficient.